### ISSUE TITLE
CA2225 generates wrong method signature when generating named alternate to explicit and implicit operators.

### ISSUE THREAD

**Issue by ronnygunawan** [dotnet/roslyn-analyzers#3076]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v2.9.7 (Latest)

#### Diagnostic ID

[CA2225](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2225?view=vs-2019)

#### Repro steps

1. Paste following code
```cs
public class I {
    public int Value { get; }
    public I(int value) {
        Value = value;
    }
    // case 1: explicit operator
    public static explicit operator I(int value) => new I(value);

    // case 2: implicit operator
    // public static implicit operator I(int value) => new I(value);
}
```
2. Run `Operator overloads have named alternates` code fix on the `explicit` or `implicit` operator

#### Expected behavior
Code fix generates following method for both `explicit` and `implicit` operator:
```cs
public static I ToI(int value) {
    throw new NotImplementedException();
}
```

#### Actual behavior
Code fix generated:
```cs
// explicit operator named alternate:
public static I ToI(I left, I right) {
    throw new NotImplementedException();
}

// implicit operator named alternate:
public I ToI() {
    throw new NotImplementedException();
}
```
